xarray/backends/cfgrib_.py

```diff
@@ -20,7 +21,9 @@
     has_cfgrib = True
 except ModuleNotFoundError:
     has_cfgrib = False
-
+except Exception as e:
+    print("WARNING: failed to load cfgrib, backend disabled", e, file=sys.stderr)
```

Comment:
```suggestion
    warnings.warn("Failed to load cfgrib - most likely eccodes is missing. Try `import cfgrib` to get the error message")
```
I suggest using a warning here.

Reply:
Ok, what module needs to be imported for `warning`?

Reply:
`import warnings` should be enough, `warnings` is part of the standard library